tests/playwright-test/resolver.spec.ts

```diff
@@ -570,3 +570,39 @@ test('should import packages with non-index main script through path resolver',
   expect(result.output).not.toContain(`find module`);
   expect(result.output).toContain(`foo=42`);
 });
+
+test('should respect tsconfig project references', async ({ runInlineTest }) => {
+  test.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/29256' });
+
+  const result = await runInlineTest({
+    'playwright.config.ts': `export default { projects: [{name: 'foo'}], };`,
+    'tsconfig.json': `{
+      "files": [],
+      "references": [
+        { "path": "./tsconfig.app.json" },
```

Comment:
I don't see `tsconfig.app.json` in this test. Intentional?

Reply:
Yes, I want to make sure we don't break on non-existing references.